apps/services/endorsements/api/src/app/modules/pagination/paginate.ts

```diff
@@ -0,0 +1,211 @@
+// SEQUELIZE CURSOR PAGINATION
```

Comment:
This file should be added centrally in the repo for other apis to use when implementing pagination. I would suggest to add it to a new library named `nest/pagination`

Reply:
my pagination implementation is pretty much a clone of https://github.com/Kaltsoon/sequelize-cursor-pagination - seeing how we want cursor based pagination which is unfortunately missing from the sequelize orm this library is the missing piece and pretty much the best thing i could find out there after digging around alot - so that being said maybe we can go as far and install this package into our environment? that would make future code cleaner and smaller,  if not i would just suggest cramming "sequelize" into the name , so something like nest/sequelize-curosr-based-pagination

Reply:
Yes we should do that instead. I thought your reason for cloning it was missing typescript definitions, but I see that they exists.

Reply:
ok - so what the best way to go about that? do i install it and commit to this PR ?

Reply:
on second thought - lets go with the first proposal - my version of the code is leaner imo

Reply:
Looks good! Just two minor comments